osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -94,7 +92,7 @@ private void load(RulesetStore rulesets)
                                                 {
                                                     new OsuSpriteText
                                                     {
-                                                        Text = $"{Score.Beatmap?.DifficultyName}",
+                                                        Text = $"{Score.Beatmap.AsNonNull().DifficultyName}",
```

Comment:
The previous coalescence never made sense here. `ScoreBeatmapMetadataContainer` was never null-tolerant and didn't check this. So I just enforced non-null in both places.